manual/source/datarules.rst

```diff
@@ -40,45 +41,52 @@ described by the following :index:`rules <rules; naming>`:
    This is important when not using the NAPI to either read or write
    the HDF5 data file.
 
-.. compound::
+.. rubric:: NXDL group and field names
 
-   .. _RegExpName:
-   
-   .. rubric:: Regular expression pattern for NeXus group and field names
+.. compound::
    
-   It is recommended that all group and field names 
-   contain only these characters:
+   The names of NeXus *group* and *field* items must 
+   contain only these characters (the names *must* match 
+   the regular expression below):
    
-   * lower case letters
-   * digits
-   * "_" (underscore character)
+   * underscore (``_``), period (``.``), digit, or letter (upper or lower case)
```

Comment:
latin alphabet letter

Reply:
revision coming to this entire block ... please stand by